From 1251c246cf7512367137f7e62d92910af7ba0ab0 Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Tue, 26 Feb 2019 18:35:07 +0000
Subject: [PATCH] TOOLS-2188 bits-upload.sh should pass --compression=none for
 uncompressed image files Reviewed by: Trent Mick <trentm@gmail.com> Approved
 by: Trent Mick <trentm@gmail.com>

---
 deps/eng | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/eng b/deps/eng
index 7c472bc..cfd827a 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit 7c472bc495ba44ac4cdde0af50ff627021942524
+Subproject commit cfd827a5029c249624efd6fcdee128db2a9d727e
-- 
2.21.0

